storage/spider/spd_param.cc

```diff
@@ -3328,6 +3328,33 @@ uint spider_param_table_crd_thread_count()
 }
 #endif
 
+static int spider_slave_trx_isolation;
+/*
+ -1 :off
+  0 :read uncommitted
+  1 :read committed
+  2 :repeatable read
+  3 :serializable
+ */
+static MYSQL_SYSVAR_INT(
+  slave_trx_isolation,
+  spider_slave_trx_isolation,
+  PLUGIN_VAR_RQCMDARG,
+  "Transaction isolation level when Spider table is used by slave SQL thread",
+  NULL, /* check */
+  NULL, /* update */
+  -1, /* def */
+  -1, /* min */
+  3, /* max */
+  0 /* blk */
+);
+
+int spider_param_slave_trx_isolation()
+{
+  DBUG_ENTER("spider_param_slave_trx_isolation");
```

Comment:
DBUG_ENTER/RETURN for such a small function not needed.

Reply:
I would like to know every function is called. So I wrote DBUG_ENTER/RETURN for all functions of Spider. I wouldn't like to remove it. Would you please approve it?

Reply:
Okay.